src/Data/RFC5322.hs

```diff
@@ -270,42 +270,40 @@ buildPhrase "" = "\"\""
 buildPhrase s =
   case enc s of
     PhraseAtom -> T.encodeUtf8Builder s
-    PhraseQuotedString _ -> "\"" <> T.encodeUtf8BuilderEscaped escPrim s <> "\""
+    PhraseQuotedString -> qsBuilder False
+    PhraseQuotedStringEscapeSpace -> qsBuilder True
```

Comment:
A yes.. I was wondering whether you use a new constructor to make it more explicit ...